.gitignore

```diff
@@ -1,2 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Not sure why we need all this on the `.gitignore` file

Reply:
Cleaden-up in 5e75fee394bee600d0207f1ec14506d8ff7b4b39